tutorials/asr/05_Online_Noise_Augmentation.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "05_Online_Noise_Augmentation.ipynb",
+      "provenance": [],
+      "collapsed_sections": [],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "cvXwyS263AMk",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "\"\"\"\n",
+        "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+        "\n",
+        "Instructions for setting up Colab are as follows:\n",
+        "1. Open a new Python 3 notebook.\n",
+        "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+        "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+        "4. Run this cell to set up dependencies.\n",
+        "\"\"\"\n",
+        "# If you're using Google Colab and not running locally, run this cell.\n",
+        "\n",
+        "## Install dependencies\n",
+        "!pip install wget\n",
+        "!apt-get install sox libsndfile1 ffmpeg\n",
+        "!pip install unidecode\n",
+        "\n",
+        "# ## Install NeMo\n",
+        "!python -m pip install --upgrade git+https://github.com/NVIDIA/NeMo.git@candidate#egg=nemo_toolkit[asr]\n",
+        "\n",
+        "## Install TorchAudio\n",
+        "!pip install torchaudio>=0.6.0 -f https://download.pytorch.org/whl/torch_stable.html\n",
+        "\n",
+        "## Grab the config we'll use in this example\n",
+        "!mkdir configs"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Kqg4Rwki4jBX",
+        "colab_type": "text"
+      },
+      "source": [
+        "# Introduction\n",
+        "\n",
+        "Data augmentation is a useful method to improve the performance of models which is applicable across multiple domains. Certain augmentations can also substantially improve robustness of models to noisy samples. \n",
+        "\n",
+        "In this notebook, we describe how to construct an augmentation pipeline inside [Neural Modules (NeMo)](https://github.com/NVIDIA/NeMo), enable augmented training of a [MatchboxNet model](https://arxiv.org/abs/2004.08531) (based on QuartzNet from the paper [\"QuartzNet: Deep Automatic Speech Recognition with 1D Time-Channel Separable Convolutions\"](https://arxiv.org/abs/1910.10261)) and finally how to construct custom augmentations to add to NeMo.\n",
```

Comment:
Minor issue: This line doe snot show up correctly on colab. I had to introduce a space between the brace and "based": 

> In this notebook, we describe how to construct an augmentation pipeline inside [Neural Modules (NeMo)](https://github.com/NVIDIA/NeMo), enable augmented training of a [MatchboxNet model](https://arxiv.org/abs/2004.08531 ) ( based on QuartzNet, from the paper ["QuartzNet: Deep Automatic Speech Recognition with 1D Time-Channel Separable Convolutions"](https://arxiv.org/abs/1910.10261)) and finally how to construct custom augmentations to add to NeMo.

Reply:
Cool sounds good. Ill update here and then merge

Reply:
Done